### PR TITLE
Header section for mobile-first-approach-based Portfolio Website

### PR DESCRIPTION
**Pull request by ProgramKingAbel**
1. Linters Set up and configured. 
-Lighthouse
-StyleLint
-WebHouse
2. Added images from Figma to the images folder
3. Detailed and Styled Markup for Header Section for mobile using HTML & CSS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,24 +24,28 @@
 # 📖 [Hello-Microverse] <a name="about-project"></a>
```
Comment (Abenezer-Tilahun) [inline README.md:24]: - [ ] Kindly update the headline of this section , it should be just like current project name instead of the "Hello-Microverse".

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @ProgramKingAbel   , 

Great job for implemented  the some of the project requirement  , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct Git Hub-Flow.
✔️ Good commit messages
✔️ Professional  README 



almost

## Required Changes ♻️
_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -4,10 +4,51 @@
...
+    <div class="wrapper">
+      <header>
+        <!--Header Logo -->
+        <div class="logo">
+          <img src="./images/Logo Placeholder.svg">
+        </div>
+        <!-- Hamburger menu -->
+        <div class="menu">
+          <img src="./images/Enabled.svg">
+        </div>
```
[the remark this answers is not in the document]
Comment (ProgramKingAbel) [reply index.html:21]: Thank you fot the reminder. Done as recommended.

File: index.html [hunk 3]
```diff
@@ -4,10 +4,51 @@
...
+
+        <div class="social-media">
+          <p class="social-media-text">LET'S CONNECT</p>
+          <div class="social-media-icons">
+            <ul>
+            <li><a href="#"><img src="./images/twitter icon.svg"></a></li>
+            <li><a href="#"><img src="./images/Linkedin icon.svg"></a></li>
+            <li><a href="#"><img src="./images/M icon.svg"></a></li>
+            <li><a href="#"><img src="./images/Chill icon.svg"></a></li>
+          </ul>
```
[the remark this answers is not in the document]
Comment (ProgramKingAbel) [reply index.html:46]: Thank you for the kind reminder. I have it fixed now.

File: index.html [hunk 4]
```diff
@@ -4,10 +4,51 @@
...
+        <div class="menu">
+          <img src="./images/Enabled.svg">
+        </div>
+      </header>
+
+      <section>
+      <div class="content">
+
+        <div class="first-headline">
+          <h2>I'm Matunda. Glad to see you!</h2>
```
[the remark this answers is not in the document]
Comment (ProgramKingAbel) [reply index.html:28]: Done as recommended.

File: index.html [hunk 5]
```diff
@@ -4,10 +4,51 @@
...
   </head>
   <body>
-    <h1>Hello Microverse!</h1>
+    <!--container wrapper-->
+    <div class="wrapper">
+      <header>
+        <!--Header Logo -->
+        <div class="logo">
+          <img src="./images/Logo Placeholder.svg">
+        </div>
```
[the remark this answers is not in the document]
Comment (ProgramKingAbel) [reply index.html:17]: changed as per your recommendation. Thank you for the remainder

File: index.html [hunk 6]
```diff
@@ -4,10 +4,51 @@
...
+
+        <div class="heading-description">
+          <p>I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
+        </div>
+
+        <div class="social-media">
+          <p class="social-media-text">LET'S CONNECT</p>
```
[the remark this answers is not in the document]
Comment (ProgramKingAbel) [reply index.html:39]: I appreciate your guidance

**Pull request merged**